packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagegroup/_mixins.scss

```diff
@@ -0,0 +1,30 @@
+@mixin chat-messagegroup(
+  $gap,
+  $padding-top,
+  $start-template,
+  $information-font-size,
+  $information-color,
+  $author-name-gap
+) {
+  .dx-chat-message-group {
+    row-gap: $gap;
+    
+    &:not(&:first-child) {
```

Comment:
Instead of this better try to consider `.dx-scrollable-content` element as flex container and use 'row-gap' instead

Reply:
This is a deliberate change because it determines how scrollable will work. In the case of gap, scrollable will not be able to properly scroll the area to the last message.

Reply:
[b5bbab3e3d2663f50b15f1c50726ddb07f6b6094]

Reply:
The last element is :after element. I can't imagine how 'clearfix' technic will used in chat, for this reason let's remove after element
```
.dx-scrollable-content::after { content: none } and will try use flex.
```

Reply:
We decided not to use the gap. Instead, we started using padding to make scrollable work properly.